docs/contributing/ctn-build.md

```diff
@@ -0,0 +1,33 @@
+The `Dockerfile` supports building and cross compiling docker daemon and extra
+tools using [Docker Buildx](https://github.com/docker/buildx) and [BuildKit](https://github.com/moby/buildkit).
+A [bake definition](https://github.com/docker/buildx/blob/master/docs/reference/buildx_bake.md) named `docker-bake.hcl` is in place to ease the build process:
+
+```shell
+# build binaries for the current host platform
+# output to ./bundles/binary by default
+docker buildx bake
+
+# build binaries for the current host platform
+# output to ./bin
+BINARY_OUTPUT=./bin docker buildx bake
+
+# build dynamically linked binaries
+DOCKER_LINKMODE=dynamic docker buildx bake
+
+# build binaries for all supported platforms
+docker buildx bake binary-cross
+
+# build binaries for a specific platform
+docker buildx bake --set *.platform=linux/arm64
```

Comment:
Overrides can only be done using the `--set` flag with bake. Agree that this example is not on point. As I don't define any target here, it will build the default one (`binary`) so we could also use `--set binary.platform=linux/arm64`. You mostly define the target in the override if you're using a group of targets. In our case we don't have any group with multi targets.